TravelPro.html

```diff
@@ -0,0 +1,295 @@
+                    <div class="icon">
+                        <img src="./images/car.png">
+                    </div>
+                    <div class="descript">
+                        <p id="head">Reach Airport on Selected Date</p>
+                        <p>Lorem ipsum dolor sit, amet consectetur adipisicing elit. Ur na Totus Tempus</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+
+        <div class="cards">
+            <div class="card-parent">
+                <div>
+                    <img src="./images/girl.jpg" width="320px">
+                </div>
+                <div class="details">
+                    <p id="trip-to-greece">Trip To Greece</p>
+                    <p>14-29 June| by Robbin john</p>
+                    <span><img src="./images/leaf.png"></span>
+                    <span><img src="./images/send.png" ></span>
+                    <span><img src="./images/send.png"></span>
+                </div>
+                <div class="details2">
+                    <div>
+                    <span><img src="./images/towers.png" height="20px" width="20px"></span>
+                    <span id="people-count">24 people going</span>
+                    </div>
+                    <span><img src="./images/heart.png"></span>
+                </div>
+            </div>
+
+            <div class="card-child">
+                <div id="image">
+                    <span><img src="./images/tomb.png" height="40px" width="40px"></span>
+                </div>
+                <div id="details_rome">
+                    <small>Ongoing</small>
+                    <p id="trip-to-rome">Trip to rome</p>
+                    <p><span>40%</span> completed</p>
+                    <img src="./images/slider_blue.jpg" height="20px" width="100px">
+                </div>
+            </div>
+        </div>
+    </div>
+
+
+    
+
+    <div class="subscribe">
+        <div class="bg_one">
+            <img src="./images/back_spiral.png" height="200px">
+            <div class="bg_two">
+                <img src="./images/back_spiral.png" height="150px" width="200px">
+            </div>
+        </div>
+        <div class="subscribe-text">
+            <h1>Subscribe to get more information, latest news and other interesting offers about Cobham</h1>
+        </div>
+        <div class="subscribe-flex">
+            <div class="subscribe-flex-box">
+            <span><img id="flex-image" src="./images/mail.png" height="30px" width="30px"></span>
+            <span><input type="text" placeholder="Your Email"></span>
+        </div>
+            <span><button>Subscribe</button></span>
+        </div>
+        <div class="send_blue_top">
+            <img src="./images/send_blue.png" height="65px" width="65px">
+        </div>
+        <div class="plus_two">
+            <img src="./images/plus.png">
+        </div>
+    </div>
+
+
+
+    
+    <footer>
+        <div class="row">
```

Comment:
Consider using more descriptive names that reflect the content or function, like footer-about, footer-contact, etc.